LiskHQ/lisk
# Node gets stuck during forward tick - Closes #1357

### What was the problem?
Node get stuck during forward tick when finish a round.

### How did I fix it?
Replace `self.scope.library.db` with `self.t` in `Round.prototype.updateVotes`.

### How to test it?
Run node, run for example `functional/http/post/0.transfer tests`, wait for node to finish a round - it should pass.

### Notice
Depends on https://github.com/LiskHQ/lisk/issues/1187 (PR https://github.com/LiskHQ/lisk/pull/1346) which should go first.

### Review checklist

* The PR solves https://github.com/LiskHQ/lisk/issues/1357
* All new code is covered with unit tests
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
